app/controllers/api/v0/sessions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V0::SessionsController < ApplicationController
+
+  def create
+    # require 'pry';binding.pry
+    user = User.find_by(email: user_params[:email])
+    if user
+      if user.authenticate(user_params[:password])
+        render json: UserSerializer.new(user), status: 200
+      else
+        render json: { error: "Bzzt, wrongo. Please try again."}, status: 401
+      end
+    else
+      render json: { error: "Bzzt, wrongo. Please try again."}, status: 401
+    end
+  end
+
+  # No idea why I'm getting a 404 sometimes or it's returning as successful? 
```

Comment:
Keeping this stuff for now. Might be able to find a solution, but if not I'll remove and clean this up before submission.